docs/index.md

```diff
@@ -80,12 +80,9 @@ the old one.
 
 ## Schema
 
-### Required
-
-- `control_plane` (String) Control plane host and API port (ex: `tenant.app.cyral.com`)
-
 ### Optional
 
 - `client_id` (String, Sensitive) Client id used to authenticate against the control plane. Can be ommited and declared using the environment variable `CYRAL_TF_CLIENT_ID`.
 - `client_secret` (String, Sensitive) Client secret used to authenticate against the control plane. Can be ommited and declared using the environment variable `CYRAL_TF_CLIENT_SECRET`.
+- `control_plane` (String) Control plane host and API port (ex: `tenant.app.cyral.com`)
```

Comment:
I believe this change was auto-generated, which is weird because it is incorrect. The `control_plane` is actually defined as a `required` field in the provider schema, in the `provider.go` file. My guess is that this could be happening due to the `DefaultFunc` used for this field in order for it to work with env vars as well, but Im not sure. Is also strange the fact that this doesn't seem to happen when running the tfplugindocs using docker.

Reply:
I actually tried to run `tfplugindocs generate` locally (`0.16.0`) and it actually generated the index.md correctly (with the `control_plane` as a required field), so Im not exactly sure what caused this change here.